multidict/_multilib/istr.h

```diff
@@ -146,34 +146,11 @@ static PyType_Spec istr_spec = {
 static inline PyObject *
 IStr_New(mod_state *state, PyObject *str, PyObject *canonical)
 {
-    PyObject *args = NULL;
-    PyObject *kwds = NULL;
-    PyObject *res = NULL;
-
-    args = PyTuple_Pack(1, str);
-    if (args == NULL) {
-        goto ret;
+    PyObject *res = PyObject_Str(str);
```

Comment:
This won't work because we need a new object. Got side tracked and haven't been able to come back to this yet